docs/tutorials/contributing/strategy/writing_the_new_strategy.rst

```diff
@@ -7,7 +7,7 @@ Identify a new strategy
 The library has a large number of strategies::
 
     >>> import axelrod as axl
-    >>> len(axl.strategies)
+    >>> len(axl.strategies) # doctest: +SKIP
     103
 
 If you're not sure if you have a strategy that has already been implemented
```

Comment:
Creates failures on travis? Yes indeed, this is non trivial...

Obviously one way is to ask for a rebase but that's not super accessible...